pkg/agent/datamodel/sig_config.go

```diff
@@ -593,11 +681,12 @@ func getSigUbuntuImageConfigMapWithOpts(opts ...SigImageConfigOpt) map[Distro]Si
 		AKSUbuntuEgressContainerd2204Gen2:  SIGUbuntuEgressContainerd2204Gen2ImageConfigTemplate.WithOptions(opts...),
 	}
 }
+
 func getSigCBLMarinerImageConfigMapWithOpts(opts ...SigImageConfigOpt) map[Distro]SigImageConfig {
 	return map[Distro]SigImageConfig{
 		AKSCBLMarinerV1:           SIGCBLMarinerV1ImageConfigTemplate.WithOptions(opts...),
 		AKSCBLMarinerV2:           SIGCBLMarinerV2Gen1ImageConfigTemplate.WithOptions(opts...),
-		AKSCBLMarinerV2Gen2:       SIGCBLMarinerV2ImageConfigTemplate.WithOptions(opts...),
```

Comment:
@hbeberman Didn't we try to changing this in the past?

Reply:
I think we left this unchanged since we didn't want to take on any unnecessary risk when we went to add in the Gen1 images in a bit of a hurry. 

Looking at this change it seems alright, and we're already going to be checking the image selection with heightened scrutiny once this merges, so now seems like a good time to correct the consistency.